kong/clustering/services/sync/rpc.lua

```diff
@@ -277,7 +277,10 @@ local function do_sync()
               ", version: ", delta_version,
               ", type: ", delta_type)
 
-      ev = { delta_type, crud_event_type, delta_entity, old_entity, }
+      -- wipe the whole lmdb, should not have events
+      if not wipe then
+        ev = { delta_type, crud_event_type, delta_entity, old_entity, }
+      end
```

Comment:
I think we could move the above assignment of `crud_event_type` inside this `if not wipe then` block. (move Line 260 inside 280~283)

Reply:
done.